cmd/spiffe-aws-assume-role/cli/cli.go

```diff
@@ -42,42 +50,63 @@ func (c *CredentialsCmd) Run(context *CliContext) error {
 		context.STSProvider,
 		c.STSEndpoint)
 	if err != nil {
-		return err
+		return errors.Wrap(err, "failed to instantiate credentials provider")
 	}
 
 	creds, err := processcreds.SerializeCredentials(provider)
 	if err != nil {
-		return err
+		return errors.Wrap(err, "failed to serialize credentials")
 	}
 
 	_, err = fmt.Print(string(creds))
 	return err
 }
 
+func (c *CredentialsCmd) configureLogger(logger *logrus.Logger) {
+	if len(c.LogFilePath) > 0 {
+		file, err := os.OpenFile(c.LogFilePath, os.O_CREATE|os.O_WRONLY|os.O_APPEND, 0666)
+		if err != nil {
+			logger.Info(errors.Wrapf(err, "Failed to log to file %s, using default of stderr", c.LogFilePath))
+		} else {
+			logger.Out = io.MultiWriter(os.Stderr, file)
+		}
+	}
+}
```

Comment:
Our new behavior:

1. Create a default Logger instance that emits text-formatted logs to stderr
2. If the --log-file-path parameter is specified, also emit logs to that file

We accomplish this by updating the configuration of the passed-in Logger after parsing is complete.